purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1182,6 +1177,11 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
          * being used
          */
         @JvmStatic var platformFlavor = "native"
+        /**
+         * DO NOT MODIFY. This is used internally by the Hybrid SDKs to indicate which platform version
+         * is being used
+         */
+        @JvmStatic var platformFlavorSDKVersion: String? = null
```

Comment:
I might combine this one and the previous one into one single static variable

Reply:
sounds good. does this have to be var? would val work? also, does it have to be public? could it be set exclusively by the constructor instead?

Reply:
The alternative to this is to set it as one of the `configure` parameters. What do you think? It needs to be public so that it can be called from the hybrid SDKs

Reply:
that was the approach for iOS, leveraging protected headers. I don't suppose there's a good equivalent? 
Could we add an extension in hybrid common that adds the constructor?

Reply:
Adding an extension in hybrid common won't make any difference since it won't have access to anything that's not public

Reply:
I see. I guess there isn't much of a way around this, then.